### PR TITLE
New release 1.2.0

### PR DESCRIPTION
**Pull request by shenitzky**
Change-Id: Ic4bf2742e248cbd3e35ab17f0029f97f783269a0
Signed-off-by: Eyal Shenitzky <eshenitz@redhat.com>

### CODE REVIEW

**Comment by shenitzky** [comment]
ci build please

File: build.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="1.1.5"
+VERSION="1.2.0"
 MILESTONE="master"
-RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+RPM_RELEASE="1.2.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
```
Comment (sandrobonazzola) [inline build.sh]: I would expect this to be 
RPM_RELEASE="1"

Comment (mwperina) [reply build.sh]: Yes, that should be a stable version

File: ovirt-ansible-disaster-recovery.spec.in [hunk 2]
```diff
@@ -37,6 +37,9 @@ sh build.sh install
 %license LICENSE
 
 %changelog
+* Thu Jan 10 2019 Eyal Shenitzky <eshenitz@redhat.com> - 1.2.0-1
+- Convert python scripts to support python3
```
Comment (mwperina) [inline ovirt-ansible-disaster-recovery.spec.in]: I'd rephrase '''Convert python scripts to support both Python 2 and Python 3'''

File: build.sh (hunk 1 above)
Comment (shenitzky) [reply build.sh]: Done

File: ovirt-ansible-disaster-recovery.spec.in (hunk 2 above)
Comment (shenitzky) [reply ovirt-ansible-disaster-recovery.spec.in]: Done

File: build.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="1.1.5"
+VERSION="1.2.0"
 MILESTONE="master"
```
Comment (sandrobonazzola) [inline build.sh]: milestone here should be empty

File: build.sh [hunk 4]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="1.1.5"
-MILESTONE="master"
-RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+VERSION="1.2.0"
```
Comment (machacekondra) [inline build.sh]: why 1.2.0 and not 1.1.5?

Comment (shenitzky) [reply build.sh]: I believe supporting python-3 gives us a new functionality, so in this case, we should bump the minor version

Comment (machacekondra) [reply build.sh]: Sure, that's up to you what versioning you decide to do. But in all other roles we are still stick to 1.1.x, so it may be confusing for someone.
